.evergreen/evergreen.yml.in

```diff
@@ -1248,7 +1250,7 @@ buildvariants:
 
   - name: pkg_smoke_tests_docker_x64
     display_name: "package smoke tests (x64 Docker)"
-    run_on: ubuntu1804-small
+    run_on: ubuntu1804-large
```

Comment:
Are you sure about these? It might be a bit wasteful to run the test-only tasks on `-large` distros, we’re probably not going to make proper use of the resources they offer compared to the `-small` ones (that’s different for build tasks, of course).

Reply:
For this I would agree that we don't need a large (maybe mistakenly included it) but the rest of the test tasks (e2es), also install nodejs and they ended up taking quite some time in my patches which I why I updated them to large.